From: Jonathan Boulle <jonathanboulle@gmail.com>
Date: Fri, 20 Jun 2014 14:46:18 -0700
Subject: fix systemd-resolved reference in man page

Guess this was missed in the copy-paste :-)

(cherry picked from commit 40ef48600f2c82c2eb41f6c71f35a8d172a530ad)
---
 man/systemd-resolved.service.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/systemd-resolved.service.xml b/man/systemd-resolved.service.xml
index ed1f5a2..517e08e 100644
--- a/man/systemd-resolved.service.xml
+++ b/man/systemd-resolved.service.xml
@@ -56,7 +56,7 @@
         <refsect1>
                 <title>Description</title>
 
-                <para><command>systemd-networkd</command> is a system
+                <para><command>systemd-resolved</command> is a system
                 service that manages network name resolution. It does so by
 		generating <filename>/run/systemd/resolve/resolv.conf</filename>,
 		which may be symlinked from <filename>/etc/resolv.conf</filename>.
